src/monster.cpp

```diff
@@ -1801,6 +1801,7 @@ void monster::melee_attack( Creature &target, float accuracy )
     }
 
     int hitspread = target.deal_melee_attack( this, melee::melee_hit_range( accuracy ) );
+    bool attack_success = hitspread >= 0;
```

Comment:
```suggestion
    const bool attack_success = hitspread >= 0;
```

use const whenever possible